js/controller.js

```diff
@@ -1623,7 +1623,8 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
       // override data in skin config with possible local storage settings, inline data input by user, and CMS settings in backlot/themebuilder
       var mergedMetaData = DeepMerge(SkinJSON, metaDataSettings, {
         arrayMerge: Utils.arrayDeepMerge.bind(Utils),
-        arrayUnionBy: 'name'
+        arrayUnionBy: 'name',
+        buttonArrayFusion: 'prepend'
```

Comment:
Do we need to save it in constants? Or it is only single using?

Reply:
It is an option for the inner arrayDeepMerge but I don't feel like we need to save it in constant. Actually I think the better way is to receive the `buttonArrayFusion` param from the metadata (looks like it was conceived initially), but it hasn't been implemented. So looks like it is a workaround.